pixels-planner/src/main/java/io/pixelsdb/pixels/planner/coordinate/StageCoordinator.java

```diff
@@ -198,7 +201,7 @@ public int getWorkerIndex(long workerId)
     /**
      * Block and wait for all the workers on this stage to ready.
      */
-    void waitForAllWorkersReady()
+    public void waitForAllWorkersReady()
```

Comment:
Why change it to public?

Reply:
Need to use this in operator. In my implementation of stream operators, workers are bootstrapped in a push-based order; e.g. in PartitionedJoinStreamOperator, its `execute()` first bootstraps all join workers, and then uses this method to wait for them to be ready (i.e. listening on their ports), and then bootstraps all partition workers.